drivers/counter/counter_mcux_rtc.c

```diff
@@ -133,20 +133,28 @@ static int mcux_rtc_cancel_alarm(struct device *dev, u8_t chan_id)
 	return 0;
 }
 
-static int mcux_rtc_set_top_value(struct device *dev, u32_t ticks,
-				  counter_top_callback_t callback,
-				  void *user_data)
+static int mcux_rtc_set_top_value(struct device *dev,
+				  const struct counter_top_cfg *cfg)
 {
 	const struct counter_config_info *info = dev->config->config_info;
+	const struct mcux_rtc_config *config =
+			CONTAINER_OF(info, struct mcux_rtc_config, info);
 	struct mcux_rtc_data *data = dev->driver_data;
 
-	if (ticks != info->max_top_value) {
-		LOG_ERR("Wrap can only be set to 0x%x", info->max_top_value);
+	if (cfg->ticks != info->max_top_value){
+		LOG_ERR("Wrap can only be set to 0x%x. "
+			"Counter reset is not supported.", info->max_top_value);
```

Comment:
The second part of the message is no longer valid.

Reply:
fixed